microsoft/testsuites/core/storage.py

```diff
@@ -85,7 +85,7 @@ class Storage(TestSuite):
         """,
         priority=2,
         requirement=simple_requirement(
-            supported_platform_type=[AZURE],
+            supported_platform_type=[AZURE, HYPERV],
```

Comment:
this case depends on waagent, it is not a hyperv case

Reply:
Fixed